Localization/autocommand.txt

```diff
@@ -0,0 +1,11 @@
+0,0
```

Comment:
what is the use of this file? Please delete if not used, or put inside a "inputs" folder.